.github/workflows/integration-test-with-multus.yaml

```diff
@@ -31,14 +31,26 @@ jobs:
         id: charm-path
         run: echo "charm_path=$(find built/ -name '*.charm' -type f -print)" >> $GITHUB_OUTPUT
 
+      - name: Setup Canonical K8S
+        id: canonical-k8s
+        run: |
+          sudo snap install k8s --classic --channel=1.32-classic/stable
+          cat << EOF | sudo k8s bootstrap --file -
+          containerd-base-dir: /opt/containerd
+          EOF
+          sudo k8s enable network dns load-balancer local-storage
+          sudo k8s status --wait-ready --timeout 5m
+          mkdir -p ~/.kube
+          sudo k8s config > ~/.kube/config
+          echo "kubeconfig=$(sudo k8s config | base64 -w 0)" >> $GITHUB_OUTPUT
+          sudo k8s set load-balancer.cidrs="10.0.0.2-10.0.0.10" load-balancer.enabled=true load-balancer.l2-mode=true
```

Comment:
Just for the sake of keeping things in order, can we move this line under the one that enables the LB? ;)